dali/dfuplus/main.cpp

```diff
@@ -168,6 +168,7 @@ void handleSyntax()
     out.append("        srcpassword=<password-for-source-dali>\n");
     out.append("    status options:\n");
     out.append("        wuid=<dfu-workunit-id>\n");
+    out.append("        limit=<limit> - For publisher wuids, maximum number of child workunits to list\n");
```

Comment:
Should we mention that the default is 1000?

Reply:
@wangkx Agree, I've made the change.